Make sure browser test timeout is >> science beam timeout

The error in [jenkins-test-elife-xpub-source-repository-575](https://alfred.elifesciences.org/job/test-elife-xpub-source-repository/575/display/redirect)

```
AssertionError: expected 'Manuscript is uploading 99%' to include 'Replace'
```

... is because the timeout for waiting for the message is a lot smaller than the timeout for sciencebeam. This should be bigger as we don't care about sciencebeam.

--- a/test/helpers/navigationHelper.js
+++ b/test/helpers/navigationHelper.js
@@ -137,7 +137,7 @@ class NavigationHelper {
   }
 
   async fillDetailsPage(minimal = true) {
-    await this.submissionPage.writeTitleIfEmpty()
+    await this.submissionPage.writeTitle()
     await this.submissionPage.selectArticleType()
     await this.submissionPage.selectSubjectAreas()
     if (!minimal) {

--- a/test/pageObjects/filesPage.js
+++ b/test/pageObjects/filesPage.js
@@ -1,3 +1,4 @@
+import config from 'config'
 import { files } from './'
 
 export class filesPage {
@@ -9,7 +10,9 @@ export class filesPage {
     'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas semper ante sed volutpat tincidunt.\n\n Nullam rutrum tortor in libero cursus, sit amet dictum ex consectetur. In eget quam ac felis suscipit sodales euismod ac urna. Donec varius mollis sapien ac pharetra. Sed non nunc neque.\n\n Aenean at lorem nisi. Etiam tempor, turpis vitae fringilla sodales, ante felis posuere eros, et imperdiet ante nisi vel tellus.'
   shortText = 'Lorem ipsum'
 
-  opts = { timeout: 6000 }
+  sbTimeout = config.get('scienceBeam.timeoutMs')
+  // This timeout should be bigger than the sciencebeam timeout (20s)
+  opts = { timeout: this.sbTimeout + 5000 }
 
   async writeCoverLetter(text = this.shortText) {
     await this.t.typeText(files.editor, text)

--- a/test/pageObjects/submissionPage.js
+++ b/test/pageObjects/submissionPage.js
@@ -17,19 +17,14 @@ export class submissionPage {
     'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas semper ante sed volutpat tincidunt.\n\n Nullam rutrum tortor in libero cursus, sit amet dictum ex consectetur. In eget quam ac felis suscipit sodales euismod ac urna. Donec varius mollis sapien ac pharetra. Sed non nunc neque.\n\n Aenean at lorem nisi. Etiam tempor, turpis vitae fringilla sodales, ante felis posuere eros, et imperdiet ante nisi vel tellus.'
   shortText = 'Lorem ipsum'
 
+  title = 'The Relationship Between Lamport Clocks and Interrupts Using Obi'
+
   discussionText = 'Spoke to Bob about another article'
   firstTitle = 'A title'
   secondTitle = 'Another Title'
 
-  async writeTitle(text = this.shortText) {
+  async writeTitle(text = this.title) {
     await this.t.selectText(submission.title).typeText(submission.title, text)
-  }
-
-  async writeTitleIfEmpty(text = this.shortText) {
-    const textContent = await submission.title.value
-    if (!textContent || textContent.length === 0) {
-      await this.writeTitle(text)
-    }
   }
 
   async selectArticleType(option = this.articleType.featureArticle) {

